src/test/java/kz/toko/app/controller/UsersIT.java

```diff
@@ -77,4 +84,29 @@ void deleteAbsentUser() throws Exception {
                 .andExpect(status().isNotFound())
                 .andExpect(content().contentType(APPLICATION_PROBLEM_JSON));
     }
+
+    @Test
+    @DisplayName("Should update only user name")
+    void updateUser() throws Exception {
+        final var updateUserRequest = new UpdateUserRequest()
+                .username("updated_username");
+
+        this.mockMvc.perform(
+                patch("/users/{id}", 1)
+                        .with(validJwtToken())
+                        .contentType(APPLICATION_JSON)
+                        .content(mapper.writeValueAsString(updateUserRequest)))
+                .andDo(print())
+                .andExpect(status().isNoContent());
+
+        this.mockMvc.perform(
+                get("/users/{id}", 1)
+                        .with(validJwtToken())
+                        .contentType(APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.id", is(1)))
+                .andExpect(jsonPath("$.username", is("updated_username")))
+                .andExpect(jsonPath("$.firstName", is("Adam")));
```

Comment:
I would suggest adding verification of the other properties too: `lastName`, `email`, `phone`